testing/downloadLivy.sh

```diff
@@ -17,12 +17,20 @@
 #
 
 if [[ "$#" -ne 1 ]]; then
-    echo "usage) $0 [livy version]"
+    echo "usage) $0 <livy version> [scala version]"
```

Comment:
usually, `<arg>` represent the required arg, but `[arg]` represent the optional arg